tools/refguide_check.py

```diff
@@ -854,6 +858,57 @@ def init_matplotlib():
         HAVE_MATPLOTLIB = False
 
 
+def check_dist_keyword_names():
+    # Look for collisions between names of distribution shape parameters and
+    # keywords of distribution methods. See gh-5982.
+    distnames = set(distdata[0] for distdata in distcont + distdiscrete)
+    mod_results = []
+    for distname in distnames:
+        dist = getattr(stats, distname)
+        if not dist.shapes:  # no collision if there are no shape parameters
+            continue
+        shape_names = dist.shapes.split(', ')
```

Comment:
This should be moved back down if we want to check for errors in documentation (in addition to collisions)